README.md

```diff
@@ -80,6 +80,18 @@ Build Status
 Thanks to GitHub Actions, all commits of this repository are tested.
 The current build status is: ![Build Status](https://github.com/SOM-st/TruffleSOM/actions/workflows/ci.yml/badge.svg)
 
+YourKit
+------------
+This Project makes use of the Yourkit profiler to help find performance bugs and aid in our research of profilers.
+
+YourKit supports open source projects with innovative and intelligent tools
+for monitoring and profiling Java and .NET applications.
+YourKit is the creator of <a href="https://eur01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.yourkit.com%2Fjava%2Fprofiler%2F&data=05%7C01%7CH.Burchell%40kent.ac.uk%7C8ae706181ac548da7e6a08db8fa58754%7C51a9fa563f32449aa7213e3f49aa5e9a%7C0%7C0%7C638261713327144205%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=fp8sUig2SCb9Z9FtOQs08vSqTWZ1BPE6joH4v9AAttU%3D&reserved=0">YourKit Java Profiler</a>,
```

Comment:
Hm, looks like you used the Outlook mangled links here instead of the plain direct ones. So this means, they encode your email address and always go through Microsoft. Could you replace them with plan links?

Reply:
should be resolved in https://github.com/smarr/TruffleSOM/pull/17/commits/a09c387565d04a0a8b1b80e1c066bdbd000a45de